src/rpc/Errors.h

```diff
@@ -50,9 +50,12 @@ struct Status
     CombinedError code = RippledError::rpcSUCCESS;
     std::string error = "";
     std::string message = "";
+    std::optional<boost::json::object> extraInfo;
 
     Status() = default;
     /* implicit */ Status(CombinedError code) : code(code){};
+    Status(CombinedError code, boost::json::object&& extraInfo)
+        : code(code), extraInfo(extraInfo){};
```

Comment:
I think you still need to `std::move` here as `extraInfo` is an lvalue here too.